### DISCUSSION TITLE
Community meeting on 23rd May

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#1821] [Community meetings]
We'll have a community meeting in about 1.5hrs, please add any discussion topics and updates to the meeting notes shared below. :)

🕒 Meeting time: 2-3 pm GMT

💻 Meeting room: [meet.google.com/rio-ngrj-kyh](https://meet.google.com/rio-ngrj-kyh)

📝 Meeting notes: https://hackmd.io/@nebari/SJ7RjXqBn

🗓️ Meeting calendar: [nebari.dev/docs/community#community-meetings](https://www.nebari.dev/docs/community#community-meetings)

**Comment by jbouder** [comment]
Any chance there are any updates on the work to run Nebari locally? In case it makes a difference I am using a MacBook, but not an M1. And I would be glad to support testing/troubleshooting if it helps accelerate the effort.

**Reply by pavithraes** [reply]
Thank you! @iameskild Do you have any thoughts or updates here?

Ref: #1405
